README.md

```diff
@@ -1,29 +1,105 @@
+</h3>
+
+
 <p align="center">
-    <img width="80" src="./img/pixelmatters-logo.jpeg" />
+SwiftLint is a tool to enforce Swift style and conventions.
 </p>
 
 <p align="center">
-    <img src="./img/pixelmatters-name.svg" />
+  <a href="https://github.com/Pixelmatters/eslint-config-pixelmatters/blob/master/LICENSE">
+    <img src="https://img.shields.io/badge/license-MIT-blue.svg" alt="eslint-config-pixelmatters is released under the MIT license." />
+  </a>
+  <a href="https://github.com/Pixelmatters/eslint-config-pixelmatters/blob/master/CONTRIBUTING.md">
+    <img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg" alt="PRs welcome!" />
+  </a>
+  <a href="https://twitter.com/intent/follow?screen_name=pixelmatters_">
+    <img src="https://img.shields.io/twitter/follow/pixelmatters_.svg?label=Follow%20@pixelmatters_" alt="Follow @pixelmatters_" />
+  </a>
 </p>
 
-<h1 align="center">
-    {PROJECT NAME}
-</h1>
 
-<h4 align="center">
-    {PROJECT DESCRIPTION}
-</h4>
+## 🚀 Installation
 
-<p align="center">
-    <a href="https://github.com/Pixelmatters/eslint-config-pixelmatters/blob/master/LICENSE">
-        <img src="https://img.shields.io/badge/license-MIT-blue.svg" alt="eslint-config-pixelmatters is released under the MIT license." />
-    </a>
-    <a href="https://github.com/Pixelmatters/eslint-config-pixelmatters/blob/master/CONTRIBUTING.md">
-        <img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg" alt="PRs welcome!" />
-    </a>
-    <a href="https://twitter.com/intent/follow?screen_name=pixelmatters_">
-      <img src="https://img.shields.io/twitter/follow/pixelmatters_.svg?label=Follow%20@pixelmatters_" alt="Follow @pixelmatters_" />
-    </a>
-</p>
+### Using [Homebrew](http://brew.sh/):
+
+Simply run the following command in your Terminal: 
+
+```
+brew install swiftlint
+```
+
+This will install the SwiftLint binaries and dependencies in your machine.
+
+### Using [CocoaPods](https://cocoapods.org):
+
+Simply add the following line to your Podfile:
+
+```ruby
+pod 'SwiftLint'
+```
+
+This will download the SwiftLint binaries and dependencies in `Pods/` during your next
+`pod install` execution and will allow you to invoke it via `${PODS_ROOT}/SwiftLint/swiftlint`
+in your Script Build Phases.
+
+This is the recommended way to install a specific version of SwiftLint since it supports
+installing a pinned version rather than simply the latest (which is the case with Homebrew).
+
+Note that this will add the SwiftLint binaries, its dependencies' binaries and the Swift binary
+library distribution to the `Pods/` directory, so checking in this directory to SCM such as
+git is discouraged.
+
+## 🛠️ Usage
+
+After to install, move the file `swiftlint.yml` into the same path of your `Project.xcodeproj`.
+
+### Xcode
+
+Integrate SwiftLint into your Xcode project to get warnings and errors displayed
+in the issue navigator.
+
+![](/Users/pauloleite/Workspace/swiftlint-config-pixelmatters/screenshot.png)
```

Comment:
To use images you need to add them into a directory (for example, `imgs`) and then use relative paths.

Reply:
My bad, I used the Typora Editor to add an image and the editor used my personal path